### DISCUSSION TITLE
问一下部分网站信息更新不出来。

### DISCUSSION THREAD

**Discussion by GodloveALei** [ronggang/PT-Plugin-Plus#1700] [Q&A]
比如我目前铃音和ub，一个是直接加入的，一个是自定义加入的，可以读取到上传下载的数据，魔力值也是正确的，但是做种体积读取不出来，一直是0。我使用了一些方法还是读取不到，请问是否是哪里配置不合理。
![photo_2023-12-31_13-01-26](https://github.com/pt-plugins/PT-Plugin-Plus/assets/114762858/b17b7485-1fa3-4692-b220-1fac86c48cad)

**Comment by hui-shao** [comment]
尝试一下使用新版。可以在 Pre-Release 或者 Action 里面下载。
目前我的是 2510 版本，Ub 是正常的。

**Reply by GodloveALei** [reply]
感谢，我之前是应用商店的所以不行，去tg下载了最新包就可以了。
